libsplinter/src/node_id/store/mod.rs

```diff
@@ -16,6 +16,7 @@
 pub mod diesel;
 /// NodeIdStore error types
```

Comment:
Not part of this PR, but this should be module doc in the error module, not a comment here.